### PR TITLE
more issue addressing

### LINKED ISSUE #2535: Simpler HILO Gold Solution
https://codeforces.com/blog/entry/97919?#comment-870891

Worth mentioning this either in the official analysis or the guide

### LINKED ISSUE #2593: Contact Form Submission - Unclear Explanation
Someone submitted the contact form!

**Module**: Bronze - Rectangle Geometry
**Topic**: Unclear Explanation
**Message**: 
The origin of the coordinate grid is not explained when talking about top right and bottom left rectangle coords - is 0,0 top left, top right, bottom left, or bottom right?

### LINKED ISSUE #2647: Contact Form Submission - Typo / Broken Link
Someone submitted the contact form!

**URL**: https://usaco.guide/adv/lagrange?lang=cpp
**Module**: Advanced - Lagrangian Relaxation
**Topic**: Typo / Broken Link
**Message**: 
Serbanology link is broken

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

should close #2593 
should also close #2647 
finally, should close #2535

### CODE REVIEW

**Review by ryanchou-dev — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
whaat the
why did you remove the solution

**Comment by SansPapyrus683** [comment]
oh ic

**Review by TruaShamu — APPROVED** [review]

File: content/2_Bronze/Rect_Geo.mdx [hunk 1]
```diff
@@ -601,19 +601,23 @@ print(rects[0].area() + rects[1].area() - intersect(rects[0], rects[2]) - inters
...
+
 Note that these formulas only apply to rectangles which have sides parallel to the coordinate axes.
 
 <Info title="Coordinates">
-A rectangle can be represented with $2$ points: its top right corner and bottom left corner. We'll label these points $tr$ (top right) and $bl$ (bottom left).
+A rectangle can be represented with $2$ points: its top right corner and bottom left corner.
+We'll label these points $tr$ (top right) and $bl$ (bottom left).
+
+Note that it doesn't really matter where you place the origin where the points are relative
+to, as long as you stay consistent.
```
Comment (bqi343) [inline content/2_Bronze/Rect_Geo.mdx:612]: ^ this comment isn't necessary imo

Comment (SansPapyrus683) [reply content/2_Bronze/Rect_Geo.mdx:612]: how 2 address #2593 then?

Comment (bqi343) [reply content/2_Bronze/Rect_Geo.mdx:612]: Say something like - In this module, we assume that increasing $x$ moves right and increasing $y$ moves up.

**Review by ryanchou-dev — APPROVED** [review]
oops sorry

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**